src/client/components/ReviewAndSubmitSection.jsx

```diff
@@ -45,28 +45,28 @@ class ReviewAndSubmitSection extends React.Component {
 
         {/* TODO(crew): Change names of sections to real names. */}
         <ReviewCollapsiblePanel
-            sectionLabel="NameAndGeneralInfoSection"
-            updatePath="/personal-information/name-and-general-information"
-            component={<NameAndGeneralInfoSection reviewSection/>}/>
+            sectionLabel="PersonalInfoSection"
+            updatePath="/veteran-information/personal-info"
```

Comment:
Some places we use `info` and some places we use `information`. Let's be consistent and use `information` everywhere?